docs/ibc/light-clients/proposal.md

```diff
@@ -1,3 +1,11 @@
 <!--
 order: 9
--->
\ No newline at end of file
+-->
+
+# Implementing `CheckSubstituteAndUpdateState`
+
+`CheckSubstituteAndUpdateState` will try to update the client with the state of the substitute client. Prior to updating, this function must verify that:
+- the substitute client is the same type as the subject client. For a reference implementation, please see the [Tendermint light client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint/proposal_handle.go#L32).
```

Comment:
we probably shouldn't link to`main` as this will change and the link will not make sense going forward. We can maybe use the most recent release?

Reply:
I had been pointing to the `02-client-refactor-beta` tag that we have. I think all of these links will need to be updated to v7 when tagged and released.